cvat/apps/engine/tests/utils.py

```diff
@@ -2,14 +2,111 @@
 #
 # SPDX-License-Identifier: MIT
 
-
-import itertools
+from contextlib import contextmanager
+from io import BytesIO
 from typing import Callable, Iterator, TypeVar
+import itertools
+import logging
+import os
 
+from django.core.cache import caches
 from django.http.response import HttpResponse
+from PIL import Image
+from rest_framework.test import APIClient, APITestCase
+import av
+import numpy as np
 
 T = TypeVar('T')
 
+
+@contextmanager
+def disable_logging():
```

Comment:
I would suggest to rename this to something like `log_disabler`, otherwise it's not clear that you're supposed to use it as a context manager.

Reply:
How about `logging_disabled`? I think, `with logging_disabled():` reads nice.

Reply:
I don't know, that name is also ambiguous, as it suggests a boolean return value. I guess it's alright.